tile[] map				# array representing silicon valley map
uint8[] target_list		# array of coordinates (array cells) of targets
coord robot_pos			# array cell containing robot
uint16 width			# in tiles, i.e. in array cells